src/client/actions/GuildMemberUpdate.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const Action = require('./Action');
+const { Status, Events } = require('../../util/Constants');
+
+class GuildMemberUpdate extends Action {
```

Comment:
```suggestion
class GuildMemberUpdateAction extends Action {
```
consistency with the other action classes